### ISSUE TITLE
Default attribute rendering not happening in body for custom categories 

### ISSUE THREAD

**Issue by treestryder** [plepe/OpenStreetBrowser#169]
I am new to creating categories and could be doing something wrong. I was hoping to benefit from the default body behavior that displays attributes in a nice way. This appears to come from using popupBody.html and/or detailsBody.html as a template. However, this does not happen when working with Custom Categories. Both, after cloning a working category or for any new custom category in a repository. 

As an example, see the "Test Shop" category in https://www.openstreetbrowser.org/dev/treestryder./paddlecraft or clone a working category. Then look at an object that would normally display details, like a shop.

**Comment by plepe** [comment]
Yes, that's right. popup and details body depend on the repository the category comes from. Custom categories are a special kind of repository, and there's no popup/details body there, so they can't be loaded/displayed.

If I want to change this behavior, I have to consider possibilities and implications. I'd rather not give a prediction on this issue (although I already have some ideas) ...

**Comment by treestryder** [comment]
Good to know. Thank you. I'll look forward to the added functionality when the category is in production. 🙂

For now, could this behavior be mentioned in the How To?

**Comment by treestryder** [comment]
For testing, could popupBody.html and detailsBody.html be copied to the custom repository?

**Comment by plepe** [comment]
No, because this is not a normal repository ...

**Comment by treestryder** [comment]
Now that they are working, I'd like to know if there is a way to suppress the things added by popupBody.html and detailsBody.html?

I like how they look. However, for some of the Paddling Amenities, like a put_in, only the description and wheelchair accessibility are rendered. So, I had created a custom body and now have them rendered twice. I would remove mine, but I was hoping to keep the description at the top, when the defaults get added to the bottom.
